new/LIT7124Theology.xml

```diff
@@ -0,0 +1,99 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7124Theology" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">Theological treatise on the nature of God</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="JK"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <listWit>
+                    <witness corresp="UppEt41"/>
+                </listWit>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>Anonymous highly abstract treatise on the nature and attributes of the Creator. In the listed manuscripts,
+                the work begins with "Chapter Three" (<foreign xml:lang="gez">አንቀጽ፡ ሣልስ፡</foreign>, 
+                    <foreign xml:lang="gez">ʾAnqaṣ śāləs</foreign>).
+                </p>
+                <p>
+                    <bibl><ptr target="bm:Loefgren1974Katalog"/><citedRange unit="page">128</citedRange></bibl>
+                    argues that it is probably a translation from Arabic, pointing to a quotation from 
+                    <ref type="author" corresp="PRS11194Yahya"/> (<ref type="mss" corresp="UppEt41"/>, fol. 2vc), 
```

Comment:
Should folia be marked up?

Reply:
I did it in the same way as in the qene records that we have been creating, but that does not mean that it is the best way :)

Do you know from the top of your head how to mark up folios in this context in the best way? I was looking for it in the guidelines but probably searched not well enough.

Reply:
I think, it can be included in the markup in the following way:
`<ref type="mss" corresp="UppEt41">Uppsala, O Etiop. 41, fol. 2vc</ref>`
I guess, the folios will be visible in search results then.

Reply:
I haven't found it in the guidelines either. `corresp` generally works with hands and other specific parts of a quoted manuscript, but I am not sure if it can work with folia too (https://betamasaheft.eu/Guidelines/?q=ref&start=16&id=references). If there's a way, the guidelines could be updated.

Reply:
I would go with what @CarstenHoffmannMarburg suggests, one could use  `UppEt41#2v` in ref but it would not be automatically expanded to read folia, and for the moment at least we have nothing that allows one to jump to a particular page in images either, so 
<ref type="mss" corresp="UppEt41">Uppsala, O Etiop. 41, fol. 2vc</ref> is sufficient or
<ref type="mss" corresp="UppEt41#2v">Uppsala, O Etiop. 41, fol. 2vc</ref> in the hope that we will ever have something to go to specific pages

Reply:
We have frequently used the simpler way used by @karljonaskarlsson, I actually think it is OK as well, we need to be able to jump to a manuscript, or find the text from the manuscript, which such reference allows.

for a more precize and sensible referencing one could add `#ms_i1` (or whichever item) after the sigla